### ISSUE TITLE
Feature Request: Improve Lighthouse Accessibility Score

### ISSUE THREAD

**Issue by wordshaker** [dotnetnotts/dotnetnotts-web#100]
### Describe the feature

The site currently has a score of 70 for accessibility when Lighthouse is run against it. 

The details of improvements that can be made are shown in the lighthouse report

![image](https://user-images.githubusercontent.com/13496774/110366452-362a4680-803e-11eb-9e82-39eb4506ae73.png)

### To Get Lighthouse Score

- Open the site in Chrome
- Open developer tools
- Go to the Lighthouse tab in the dev tools
- Run.

### Advantages

- Improving the accessibility of the site

#### Acceptance Criteria

Functional and non-functional criteria that should be met by this work

- [ ] Lighthouse score for accessibility is increased to at least 90.
